src/sparql/mondo-synonyms-scope-type-xref.sparql

```diff
@@ -0,0 +1,29 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX oio: <http://www.geneontology.org/formats/oboInOwl#>
+PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+SELECT
+  ?mondo_id ?dbXref ?synonym_scope ?synonym ?synonym_type
+WHERE {
+  ?class a owl:Class .
+
+  ?axiom owl:annotatedSource ?class ;
+     owl:annotatedProperty ?synonymPropertyUri ;
+     owl:annotatedTarget ?synonym .
+
+  OPTIONAL { ?axiom oboInOwl:hasDbXref ?dbXref }
```

Comment:
## Bug fix: optional `hasDbXref`
I accidentally forgot to make this one `OPTIONAL` as well. The change has increased the number of results from about 131k to 138k; so about 7k synonyms without evidence.

FYI @twhetzel @matentzn